Mathlib/Order/SuccPred/Limit.lean

```diff
@@ -252,6 +252,17 @@ theorem isSuccPrelimit_iff_succ_lt : IsSuccPrelimit b ↔ ∀ a < b, succ a < b
 @[deprecated isSuccPrelimit_iff_succ_lt (since := "2024-09-05")]
 alias isSuccLimit_iff_succ_lt := isSuccPrelimit_iff_succ_lt
 
+theorem coe_succ_Iio {a : α} (h : IsSuccPrelimit a)
```

Comment:
I think it's fine for `x` to be explicit. It should give you strictly more flexibility in `rw` or `have` without making things worse in the cases where having `x` implicit already works.